### ISSUE TITLE
[Settings]: Name of 'Tab Title' edit field is not defined.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12022]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

Repro Steps:
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Navigate to Settings> Action> Add new button.
4. Add shortcut 'Ctrl + Shift +r' for 'Rename Tab Title' actions.
5. Open Narrator using 'Win + Ctrl+ Enter' keys.
6. Save Changes using 'Save' button.
6. Then press 'Ctrl + Shift + r' to rename tab title.
7. Observe the issue.

**User Experience:**
Screen Reader users UX will not be good if they are not able to know the purpose of the edit field. 

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachments:**
[Name of 'Tab Title' edit field is not defined..zip](https://github.com/microsoft/terminal/files/7769308/Name.of.Tab.Title.edit.field.is.not.defined.zip)

### Expected Behavior

Name of 'Tab Title' edit field should be defined.
i.e. For 'Windows Terminal' tab title, Screen Reader should announce 'Rename Tab Title edit <placeholder text>'

### Actual Behavior

Name of 'Tab Title' edit field is not defined.
i.e. For 'Windows Terminal' tab title, Screen Reader is only announcing 'edit <placeholder text>'.

**Note Issue:** _Screen Reader is only announcing 'Edit placeholder' for 'Window 1 Enter a new name' edit field inside 'Windows 1' pop-up._
